### DISCUSSION TITLE
Prompt Flow help - embedding is failing!

### DISCUSSION THREAD

**Discussion by ipshitag** [community/community#65368] [Programming Help]
Hi,
So I had this flow for RAG, which embedded the question before retrieving, the embedding api is gone now 😢 What to do?
![image](https://github.com/community/community/assets/20279993/f4389698-3b12-4294-b174-35d899ab5538)

Error on chat:
![image](https://github.com/community/community/assets/20279993/99e5c7c3-47e3-444b-a15f-bd7c592833d4)

**Comment by sethjuarez** [comment]
Great question! There is now a separate embedding tool that you can use specifically for this task. Let me know if this helps!
